src/crimson/os/seastore/cache.cc

```diff
@@ -658,7 +681,12 @@ void Cache::commit_retire_extent(
   DEBUGT("extent {}", t, *ref);
   assert(ref->is_valid());
 
-  remove_from_dirty(ref);
+  // TODO: why does this duplicate remove_extent?
```

Comment:
probably commit_retire_extent() can call remove_extent() to reuse the logic?

Reply:
I think it's better not to reuse this logic for clearer debug information.